lib/datadog/core/environment/yjit.rb

```diff
@@ -52,11 +52,20 @@ def yjit_alloc_size
           ::RubyVM::YJIT.runtime_stats[:yjit_alloc_size]
         end
 
+        # Ratio of YJIT-executed instructions
+        def ratio_in_yjit
+          ::RubyVM::YJIT.runtime_stats[:ratio_in_yjit]
+        end
+
         def available?
           defined?(::RubyVM::YJIT) \
             && ::RubyVM::YJIT.enabled? \
             && ::RubyVM::YJIT.respond_to?(:runtime_stats)
         end
+
+        def stats_available?
+          available? && ::RubyVM::YJIT.respond_to?(:stats_enabled?)
+        end
```

Comment:
Minor: Since this is only for the tests to use, I think it's better to keep it there, otherwise it looks like a forgotten/abandoned method (e.g. "why doesn't `available?` call `stats_available?` comes to mind as a question when looking at this without context).